docs/guides/migrating-from-camunda-platform-7/migration-readiness.md

```diff
@@ -0,0 +1,278 @@
+
+- **Architecture:** The different architecture of the core workflow engine might require _changes in your architecture_ (e.g. if you used the embedded engine approach). Furthermore, certain concepts of Camunda Platform 7 are no longer possible (like hooking in Java code at various places, or control transactional behavior with asynchronous continuations) which might lead to _changes in your model and code_.
+
+In general, **workflow engine data** is harder to migrate to Camunda Platform 8:
+
+- **Runtime data:** Running process instances of Camunda Platform 7 are stored in the Camunda Platform 7 relational database. Like with a migration from third party workflow engines, you can read this data from Camunda Platform 7 and use it to create the right process instances in Camunda Platform 8 in the right state. This way, you can migrate running process instances from Camunda Platform 7 to Camunda Platform 8. [A process instance migration tool](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/process-instance-migration) is in place to ease this task. This tool is community supported.
+
+- **History data:** Historic data from the workflow engine itself cannot be migrated. However, data in Optimize can be kept.
+
+## Migration tooling
+
+:::note
+These tools are community maintained. For more assistance, create an issue on the repo directly.
+:::
+
+The [Camunda Platform 7 to Camunda Platform 8 migration tooling](https://github.com/camunda-community-hub/camunda-7-to-8-migration), available as a community extension, contains three components that will help you with migration:
+
+1. [A converter available in different flavors (web app, CLI) to convert BPMN models from Camunda Platform 7 to Camunda Platform 8](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/backend-diagram-converter). This maps possible BPMN elements and technical attributes into the Camunda Platform 8 format and gives you warnings where this is not possible. The result of a conversion is a model with mapped implementation details as well as hints on what changed, needs to be reviewed, or adjusted to function properly in Camunda Platform 8.
+
+2. [The Camunda Platform 7 Adapter](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/camunda-7-adapter). This is a library providing a worker to hook in Camunda Platform 7-based glue code. For example, it can invoke existing JavaDelegate classes.
+
+3. [A process instance migration tool](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/process-instance-migration) to migrate running process instances from Camunda Platform 7 to Camunda Platform 8. Ideally, you should let running instances finish prior to migrating.
+
+The tools mentioned above are a good starting point, but are only one option for how you can approach your migration.
```

Comment:
What are the other options for migration? Should we link to them?

Reply:
Essentially, this is the rest of the guide 😅 

The goal of this sentence should be to clarify that the tools aren't necessary (seeing as if they were necessary, we would productize them). With that context, do you have a suggestion for how to rework this?

Reply:
Okay -- that was my assumption. A quick fix would be:

"The tools mentioned above are a good starting point, but are only one option for how you can approach your migration, **as described below**."

Reply:
If you think most developers wouldn't really require this clarification, we can leave out.

Reply:
Will add!